### ISSUE TITLE
Unity package manager failed to automatically install the Open VR package

### ISSUE THREAD

**Issue by nitsuogg566** [ValveSoftware/steamvr_unity_plugin#393]
I got the same problem as this  https://github.com/ValveSoftware/steamvr_unity_plugin/issues/310#issuecomment-476865394
I'm also using Unity 2018.1.1f1.
Can anyone help me?

**Comment by Spikesmaster** [comment]
I've got this issue with 2018.1 as well, updating to 2018.3 (currently on 2018.3.8f1) fixed the issue for me, if you can't update to .3 try to find a newer version and see if works.

**Comment by nitsuogg566** [comment]
Thanks, @Spikesmaster  I tried upgrading to 2018.3 then got lots of shader errors...
I'm pretty much stuck here :(

**Comment by zite** [comment]
Have you tried opening the Package Manager UI manually from the Window menu?

**Comment by nitsuogg566** [comment]
@keithbradner yes! But there is no Open VR package in the Package manager.

**Comment by nyjackson** [comment]
Hi! So I updated Unity to version 2018.1.6f1, and had the same issue. I tried to find OpenVR in the Package Manager, but it wasn't listed. I came across this thread, and some others, but found no solution. So, I found the original code for the most recent OpenVR on Github via the link: https://github.com/ValveSoftware/openvr 
I downloaded the file and replaced my openvr_api.cs in my project's Assets/SteamVR/Plugins with the openvr_api.cs from the openvr-master/headers. After replacing it, I no longer get the error, and I can now play/run my project. I hope this helps!
